packages/docs-site/docs/tutorial/basics.md

```diff
@@ -85,9 +85,9 @@ forall Set x {
 }
 ```
 
-So, what are the shapes we can use? Currently, the system supports 12 different shapes, and you can find the specs for every shape [here](/docs/api/). It is a page that you will visit frequently as you work in Penrose.
+So, what are the shapes we can use? Currently, the system supports 12 different shapes, and you can find the specs for every shape [here](/docs/ref/). It is a page that you will visit frequently as you work in Penrose.
```

Comment:
Is it possible to create a parent page for "Shape Library", and link to that? Currently this just links to the root documentation page, and if someone doesn't already have "Style" expanded in the sidebar, it's unclear where they're supposed to go from here.